crates/codegen/runtime/cargo/crate/src/runtime/bindings/mod.rs

```diff
@@ -23,3 +26,7 @@ pub fn create_with_resolver(
 pub fn get_binding_rules() -> &'static str {
     binding_rules::BINDING_RULES_SOURCE
 }
+
+pub fn get_built_ins(version: &semver::Version) -> &'static str {
```

Comment:
nit suggestion:

```suggestion
#[cfg(feature = "__private_testing_utils")]
pub fn get_built_ins(version: &semver::Version) -> &'static str {
```

Reply:
We need to be able to get the built-ins to parse and add to the Bindings API externally now, so this function (or some variant of it) needs to be public. Otherwise users will not be able to configure their Bindings API properly.